### DISCUSSION TITLE
Small question

### DISCUSSION THREAD

**Discussion by Dixxi0509** [tahoma2d/tahoma2d#1177] [General]
Is there a way to get a storyboard feature and I have a question about transparency how can I do transparent and render it with the animation

**Comment by ABelliqueux** [comment]
You can see transparency by unchecking "Camera BG color" in the "View" menu.
You can then render using a format that supports transparency (mov, png, tif, gif).

If you're talking about transparency between levels, you can use layer blending or matte nodes in the FX tab.
